common/lib/xmodule/xmodule/modulestore/xml_importer.py

```diff
@@ -419,7 +419,8 @@ def run_imports(self):
                 dest_id, runtime = self.get_dest_id(courselike_key)
             except DuplicateCourseError:
                 continue
-            with self.store.bulk_operations(dest_id):
+            #with self.store.bulk_operations(dest_id):
+            if True:
                 source_courselike, courselike, data_path = self.get_courselike(courselike_key, runtime, dest_id)
```

Comment:
@cpennington  And this is the hack "fix" for the problem - I'd like to discuss this with you.